hq/markdown/snyk.md

```diff
@@ -0,0 +1,227 @@
+# Snyk
+
+## Introduction
+
+Snyk is a managed service which uses a dependency analysis tool and a vulnerability database to inform you of
+vulnerabilities in your application.
+
+## Installing
+
+Snyk requires node, and `npm`.  To install on a mac, please use:
+
+```
+brew install npm
+npm install -g snyk
```

Comment:
What's the unicode char at the start of the line here? I'm seeing a character error (box)

Reply:
God knows, but it's gone now.